src/main/java/com/coreos/jetcd/EtcdLease.java

```diff
@@ -0,0 +1,40 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.api.LeaseGrantResponse;
+import com.coreos.jetcd.api.LeaseKeepAliveResponse;
+import com.coreos.jetcd.api.LeaseRevokeResponse;
+import com.google.common.util.concurrent.ListenableFuture;
+
+/**
+ * Interface of Lease talking to etcd
+ */
+public interface EtcdLease {
+
+    /**
+     * New a lease with ttl value
+     * @param ttl ttl value
+     * @return
+     */
+    ListenableFuture<LeaseGrantResponse> grant(long ttl);
```

Comment:
The ListenableFuture is the default return type of gRPC, so I use it here. #16 have talk about this issue.